src/main/webapp/place-suggestions.html

```diff
@@ -0,0 +1,131 @@
+        </ul>
+      </div>
+    </nav>
+
+    <ul class="sidenav" id="mobile-demo">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="my-trips.html">My Trips</a></li>
+      <li class="active"><a href="#">Plan For Me</a></li>
+      <li><a href="trips-network.html">Trips Network</a></li>
+    </ul>
+
+    <div id="map"></div>
+
+    <div class="page-container container">
+      <div id="input-container">
+        <div id="city-input-area" class="row">
+          <h5 id="city-input-title" class="col s12 m7">
+            What city do you want to visit?
+          </h5>
+          <div id="city-input-field" class="input-field col s12 m5">
+            <input placeholder="Enter a city" id="city-input" type="text">
+          </div>
+        </div>
+
+        <div id="filter-selection-area" class="row">
+          <h5 id="filter-selection-title" class="col s12 m7">
+            What type of place do you want to visit?
+          </h5>
+          <div id="filter-selection" class="input-field col s12 m5">
+            <select id="filters">
+              <option value="" disabled selected>Choose place type</option>
+              <option value="amusement_park">Amusement Park</option>
+              <option value="aquarium">Aquarium</option>
+              <option value="art_gallery">Art Gallery</option>
+              <option value="bakery">Bakery</option>
+              <option value="bar">Bar</option>
+              <option value="book_store">Book Store</option>
+              <option value="bowling_alley">Bowling Alley</option>
+              <option value="cafe">Cafe</option>
+              <option value="campground">Campground</option>
+              <option value="casino">Casino</option>
+              <option value="movie-theater">Movie Theater</option>
+              <option value="museum">Museum</option>
+              <option value="night-club">Night Club</option>
+              <option value="park">Park</option>
+              <option value="restaurant">Restaurant</option>
+              <option value="shopping_mall">Shopping Mall</option>
+              <option value="spa">Spa</option>
+              <option value="stadium">Stadium</option>
+              <option value="tourist_attraction">Tourist Attraction</option>
+              <option value="zoo">Zoo</option>
+            </select>
+          </div>
+        </div>
+      </div>
+
+      <div id="place-cards-container" class="row"></div>
+    </div>
+
+    <footer class="page-footer blue lighten-1">
+      <div class="footer-copyright">
+        <div class="container">
+          Created by Peter Wu, Joshua Lewis, and Miguel Fabrigas
+        </div>
+      </div>
+    </footer>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+    <script src="scripts/constant-script.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBNsFRB5UFiW3Atam7bg32K_Dn47t4if0k&libraries=places"></script>
```

Comment:
Let's standardize the API key to be the same one from the `my-trips.html` page, since that's the API key for the actual overnightly-step27-2020 project on Google Cloud.

Reply:
Fixed